plutus-contract/src/Plutus/Contract/Request.hs

```diff
@@ -283,6 +296,17 @@ currentTime ::
     => Contract w s e POSIXTime
 currentTime = pabReq CurrentTimeReq E._CurrentTimeResp
 
+-- | Get the 'POSIXTime' range of the current slot.
+--
+-- Example: if slot length is 3s and current slot is 2, then `currentTimeRange`
+-- returns the time interval @[3, 5[@.
+currentNodeClientTimeRange ::
+    forall w s e.
+    ( AsContractError e
+    )
+    => Contract w s e (POSIXTime, POSIXTime)
```

Comment:
As the doc says interval, and as being open or closed may be a relevant info, shouldn't we return an `Interval` instead? (and probably re-export the Interval API to ease their manipulation)

Reply:
Happy to change it, but after looking at `Interval`, it seemed to me harder to manipulate an `Interval` than (POSIXTime, POSIXTme), especially when you want to get the actual lower bound of the interval. The reason is that the lowerbound can be `-Inf`, `Finite` or `Inf`. Same for the upper bound.

Thoughts?

Reply:
Oh yes you're right. It's asd that we cant have the boundaries closure to be really explicit about it, but I don't think it's sad enough to create a new type for this, so I guess we can go with a pair.